src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-solution-proposal-step.html

```diff
@@ -0,0 +1,389 @@
+<!-- PROBLEM SOLVED -->
+<div data-ng-if="$ctrl.currentLineDiagnostic.data.answers.resolvedAfterTests">
+    <img class="col-md-2" src="assets/images/diagnostic/human-good.svg" width="180" height="180" />
+    <span class="col-md-8">
+        <p class="oui-header_4"
+           data-translate="tools_lineDiagnostics_solution_proposal_step_problem_solved_title"></p>
+        <p data-translate="tools_lineDiagnostics_solution_proposal_step_problem_solved_subtitle"></p>
+        <oui-button data-variant="primary"
+                    data-text="{{:: 'tools_lineDiagnostics_solution_proposal_step_exit_diagnostic' | translate }}"
+                    data-on-click="$ctrl.$state.go('^')"></oui-button>
+    </span>
+</div>
+
+<!-- SPECIFIC ERRORS -->
```

Comment:
Lot of repetive code, could be optimized to be more generic

Reply:
Yes i could factorize, but there are many status who have specific changes (translation needed, name of translatiion, some values), and this would end up maybe with smaller code, but with many if conditions, and then this could be difficult to read